lib/awful/menu.lua.in

```diff
@@ -470,7 +470,7 @@ function menu.clients(args, item_args)
                 if not c:isvisible() then
                     tags.viewmore(c:tags(), c.screen)
                 end
-                capi.client.focus = c
+                c:emit_signal("request::activate", "focus")
```

Comment:
sorry to have missed this in my previous review, but I think "menu.clients" would be more descriptive. Other than that this PR look good. Once you fix this, if nobody complain I will go ahead and merge it.